src/main/java/luke/Deadline.java

```diff
@@ -0,0 +1,41 @@
+package luke;
+
+import java.time.LocalDate;
+
+public class Deadline extends Task {
```

Comment:
Maybe "DeadlineTask" as a class name might be more relevant and descriptive compared to simply "Deadline." Might have a Deadline task in the future. Applicable to Event & Todo as well. :)